subsys/bluetooth/services/fast_pair/fp_storage.c

```diff
@@ -61,63 +63,81 @@ static uint8_t next_key_id(uint8_t key_id)
 	return key_id + 1;
 }
 
-static int fp_settings_set(const char *name, size_t len, settings_read_cb read_cb, void *cb_arg)
+static int fp_settings_load_ak(const char *name, size_t len, settings_read_cb read_cb, void *cb_arg)
 {
 	int rc;
 	uint8_t index;
 	struct account_key_data data;
-	const char *key = SETTINGS_AK_NAME_PREFIX;
+
```

Comment:
unnecessary blank line

Reply:
Wanted to visually group variables, but anyway removed.